src/Symfony/Component/Mailer/Tests/Transport/SendmailTransportTest.php

```diff
@@ -38,10 +38,27 @@ protected function tearDown(): void
 
     public function testToString()
     {
+        $previousSendmailPath = \ini_set('sendmail_path', '');
```

Comment:
This should happen in `setUp()`.

Reply:
or we should use `$this->iniSet()`?

Reply:
👀 sebastianbergmann/phpunit#5214

Reply:
It is not relevant to all tests, though.